### ISSUE TITLE
Is it possible for Tauon to ignore .folder (hidden) during rescan?

### ISSUE THREAD

**Issue by Horuyuki** [Taiko2k/TauonMusicBox#584]
I hid some folder in the music directory because I don't want them to show up for now.

But upon rescan tauon still shows it in the playlist. Is there a way for tauon to ignore hidden folders during rescan?

**Comment by Taiko2k** [comment]
No, but it makes sense that Tauon should always ignore dot files and folders. I've committed this change now.

**Comment by Horuyuki** [comment]
> No, but it makes sense that Tauon should always ignore dot files and folders. I've committed this change now.

Thank you so much! Glad it was an easy quick fix. Will wait for your next release!
